Explains that `application/pdf` is already registered by Rails [ci skip] (#52597)
rails/rails: guides/source/action_controller_overview.md

end
```

> This example works because [Rails already registers the `"application/pdf"` MIME type](https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/http/mime_types.rb) at startup (alongside other common types). If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example:

Reviewer: We could probably remove this content altogether, as it feels out of place now, also rare to do so.

There is no documentation on https://edgeapi.rubyonrails.org/classes/Mime/Type.html#method-c-register , that is a better place to start adding this instead.

Author: I still think it's important to have that comment in there, because it can be surprising to someone seeing the `respond_to` followed by `format.pdf` and wondering "how does Rails know what to do with PDF files?", and "where does the `.pdf` method name comes from?" or even "what other formats does it support?". Those are the questions I had every time I read that part of the guides.

> This example works because [Rails already registers the `"application/pdf"` MIME type](https://github.com/rails/rails/blob/b26a9fd2551a2e65e8deef10f4b5ca6b99d666fb/actionpack/lib/action_dispatch/http/mime_types.rb) at startup (alongside other common types). If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example, this is how you would register the Rich Text Format (RTF):

Reviewer 2: I think it's more helpful to document why we need to register the MIME type.
It also helps to document how you can check which MIME types are  already supported.
Maybe something like:
````suggestion
You can call any method on `format` that is an extension registered as a MIME type by Rails.
Rails already registers common MIME types like `"text/html"` and `"application/pdf"`:
```ruby
Mime::Type.lookup_by_extension(:pdf)
# => "application/pdf"
```

If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example, this is how you would register the Rich Text Format (RTF):
````

Author: I think that's great as it ties it all together. I see what you meant about the lookup method.

```ruby

> Mime::Type.register(<mime_type>, <symbol>)

Reviewer 3: It seems like the linter expects this to be valid ruby.

Author: Perhaps the example could be removed altogether, since there's already a link to the method.

I couldn't come up with an example to use, but also didn't want to give the impression that it's super expected practice to add new MIMEs, because Rails supports so many out of the box. I've been tripped by the PDF example many times 😅 and all my apps have a `mime` file that's probably not needed at all.
